### PR TITLE
feat(client): Create BlogCard component for FrontPage

### LINKED ISSUE #77: Create Blog card component

### PR DESCRIPTION
**Pull request by karansapolia**
Create BlogCard component for FrontPage and create a dummy
data file to access from on the Home component.

fixes: #77

## PR Checklist
Please check that your PR fulfills the following requirements:

- [x] The commit message follows [our guidelines](https://github.com/pesto-students/batch-9-Lumen/wiki/Commit-message-convention)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [x] GIFs/Videos/Images added (only for frontend changes)

## PR Type
What kind of change does this PR introduce? 

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Infrastructure changes
- [ ] Other... Please describe:

## Issue Number: 
#77 

## Does this PR introduce a breaking change?
- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by karansapolia** [comment]
GIF of changes to homepage:
Peek 2019-08-22 19-05

File: client/src/containers/Home.jsx [hunk 1]
```diff
@@ -1,50 +1,42 @@
 import React from 'react';
 import styles from './Home.module.css';
-import Blog from '../components/Blog';
 import FloatingActionButton from '../components/FloatingActionButton/FloatingActionButton';
+import BlogCard from '../components/BlogCard/index';
+import topTen from '../topTen';
+
+const biggerCards = topTen.slice(0, 2).map((props) => (
```
Comment (nimish-gupta) [inline client/src/containers/Home.jsx:7]: can be use random instead of 0,2 it will be fun, on every reload it will look different

**Review by nimish-gupta — APPROVED** [review]

File: client/src/containers/Home.jsx (hunk 1 above)
Comment (karansapolia) [reply client/src/containers/Home.jsx:7]: The first two will be the top most posts always, sure, we will randomize the other posts to make it more fun! :smile:

**Review by nihalanisumit — APPROVED** [review]
LGTM!

**Review by piyushagwl007 — APPROVED** [review]
LGTM :smile: Good work :smiley:

**Comment by piyushagwl007** [comment]
@karansapolia please merge your branch if the work here is completed :smile:

**Pull request merged**